components/central-application-gateway/internal/proxy/reverseproxy.go

```diff
@@ -39,8 +40,9 @@ func newProxy(targetURL string, requestParameters *authorization.RequestParamete
 		req.Host = target.Host
 
 		combinedPath := joinPaths(target.Path, req.URL.Path)
-		req.URL.RawPath = combinedPath
+		combinedPathE := joinPaths(target.Path, req.URL.RawPath)
```

Comment:
Variable name verbosity should be proportional to scope of use, so I didn't want to name it `combinedPathEscaped` considering it's only used in the next line, but I see your point. I'll change it to the long version, but I am open to suggestions.

Reply:
True. My point was `combinedPathE` says nothing about why it's there :D By just looking at the way of creating it's logical to name it `combinedRawPath` as it would be consistent with the `42` line. I assume that it's more about the escaping rather than the rawness :D Therefore, `combinedPathEscaped` sounds fine